articles/governance/policy/how-to/remediate-resources.md

```diff
@@ -197,7 +254,7 @@ To create a **remediation task**, follow these steps:
    :::image type="content" source="../media/remediate-resources/task-progress.png" alt-text="Screenshot of the Remediation tasks tab and progress of existing remediation tasks." border="false":::
 
 1. Select on the **remediation task** from the policy compliance page to get details about the
-   progress. The filtering used for the task is shown along with a list of the resources being
+   progress. The filtering used for the task is shown along with status and a list of the top 500 resources being
```

Comment:
Same as above, I don't think it's worth mentioning since this is an edge case for large remediation and when it happens, we show an info box to the user. No need to document all of the UX edge cases if the UX is already explaining them when they happen :)

Reply:
Dropped the top 500.